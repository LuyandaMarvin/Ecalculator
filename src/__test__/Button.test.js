import React from "react";
import Button from "../components/Button"
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CalcProvider from "../context/CalcContext";

configure({ adapter: new Adapter() });

describe('<Wrapper />', () => {
  const button = shallow(
    <CalcProvider>
         <Button  value={'='} key={1} />
    </CalcProvider>
  );
  
    it('renders the correct html output', () => {
      expect(button.html()).toMatchSnapshot();
    });
});