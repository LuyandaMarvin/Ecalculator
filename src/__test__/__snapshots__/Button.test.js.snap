// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> renders the correct html output 1`] = `"<button data-testid=\\"button\\" class=\\"equals button\\">=</button>"`;
